responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -57,6 +61,9 @@ def __init__(
         self._categorical_features = categorical_features
         if categorical_features is None:
             self._categorical_features = []
+        self._dropped_features = dropped_features
```

Comment:
Super nit: if you want to shorten this you can write 

dropped_features or []

Reply:
Updated, thanks!